docs/configure

```diff
@@ -867,22 +867,10 @@ virtual_delivery:
   mode = 0600
   maildir_format = true
   create_directory = true
-  directory = ${lookup mysql{select smtp from users,domains \
-		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}}
-  user = ${lookup mysql{select users.uid  from users,domains \
-		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}}
-  group = ${lookup mysql{select users.gid from users,domains \
-		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}}
-  quota = ${lookup mysql{select users.quota from users,domains \
-  		where localpart = '${quote_mysql:$local_part}' \
-		and domain = '${quote_mysql:$domain}' \
-		and users.domain_id = domains.domain_id}{${value}M}}
+  directory = ${extract{smtp}{$address_data}}
```

Comment:
Are you sure we don't need to define `$address_data` for each router? I think we should, at the very least it would make things clearer, don't you think?